source/_integrations/motionblinds_ble.markdown

```diff
@@ -17,13 +17,14 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The Motionblinds BLE {% term integration %} adds support for [Motionblinds](https://motionblinds.com/) Bluetooth motors. Beware that this integration does not work with *Eve Motionblinds* motors. *Eve Motionblinds* can be added to Home Assistant using the [HomeKit Device](https://www.home-assistant.io/integrations/homekit_controller/) integration.
+This {% term integration %} adds support for [Motionblinds](https://motionblinds.com/) Bluetooth motors. Want to motorize your existing blinds? This can be done with Motionblinds motor CMD-03 which is available through the following stores: [Hornbach](https://www.hornbach.com/), [Coolblue](https://www.coolblue.nl/), [Proshop](https://www.proshop.com/), [Raamdecoratie.com](https://www.raamdecoratie.com/), [Robbshop.com](https://www.robbshop.nl/). Made-to-measure window coverings with Motionblinds are available through a worldwide reseller network which can be found [here](https://motionblinds.com/stores/).
```

Comment:
The purchasing information is out of the scope of the user documentation here. Please remove the links to the stores.